workflow/rules/diffexp.smk

```diff
@@ -306,6 +308,7 @@ rule plot_fragment_length_dist:
             "results/plots/fld/{sample}-{unit}.fragment-length-dist.pdf",
             caption="../report/plot-fld.rst",
             category="Fragment length distribution",
+            labels={"samples": "{sample}-{unit}"},
```

Comment:
Put this into category `category="QC"`, `subcategory="per-sample"`, and then `labels={"sample": "{sample}-{unit}", "plot": "fragment lengths"}`. In addition, put all the the model-specific QC into `subcategory="per-model"`.